jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/ContainerParameters.java

```diff
@@ -63,25 +63,29 @@ public ContainerParameters(ObjectFactory objectFactory) {
     creationTime = objectFactory.property(String.class).convention("EPOCH");
     mainClass = objectFactory.property(String.class);
     jvmFlags = objectFactory.listProperty(String.class);
+    entrypoint = objectFactory.listProperty(String.class);
   }
 
   @Input
-  @Nullable
```

Comment:
Just a similar concern here for updating nullability for this public getter: https://github.com/GoogleContainerTools/jib/pull/3936#discussion_r1156115328

Reply:
Ok I updated this to keep in Nullable

Reply:
Thanks!